spec/javascripts/fixtures/fixture.html.erb

```diff
@@ -1,4 +1,5 @@
-#fixture_view
-  - @title = "Title"
-  %h1= @title
-  %p Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur auctor, erat vel porta consectetur, erat lectus venenatis arcu, vel vehicula ipsum dui sed neque. Cras eros arcu, fermentum non semper quis, placerat sit amet metus. Mauris id mi nec tortor varius tristique sed in urna. Pellentesque vitae augue eu libero dignissim gravida sed quis neque. Aenean gravida sollicitudin nibh quis hendrerit. Donec justo dui, rutrum pharetra sodales a, fermentum quis erat. Nulla cursus, quam eget volutpat convallis, risus libero semper libero, sed commodo lorem ligula quis quam. In hac habitasse platea dictumst. Nulla facilisi. In eget dolor felis. Aliquam varius mi ac magna congue fermentum.
+<div id="fixture_view">
+  <% @title = "Title" %>
```

Comment:
Super minor nitpick: Shouldn't this line be translated as `<%- @title = "Title"  -%>` to preserve the same semantics as the HAML `- @title = "Title"`?

Reply:
technically no, `-` vs `=` in haml is if you want output of the ruby expression, or if you just want the ruby expression to run. In erb `-` is about whitespace, and since there's already whitespace here I don't think it works the way you're thinking.